extensions/iceberg/src/main/java/io/deephaven/iceberg/util/SortOrderProvider.java

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright (c) 2016-2025 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.iceberg.util;
+
+import org.apache.iceberg.SortOrder;
+
+/**
+ * A specification for providing {@link org.apache.iceberg.SortOrder} while writing to an iceberg table.
+ */
+public interface SortOrderProvider {
```

Comment:
Missing a `from(SortOrder)`

Reply:
May need both `org.apache.iceberg.SortOrder#sameOrder` and `org.apache.iceberg.SortOrder#equals` to verify, not sure.